src/platform/qt/ts/mgba-pt_BR.ts

```diff
@@ -0,0 +1,5445 @@
+      <translation>Game Boy Color (CGB)</translation>
+    </message>
+    <message>
+      <location filename="../OverrideView.ui" line="268"/>
+      <source>Game Boy Advance (AGB)</source>
+      <translation>Game Boy Advance (AGB)</translation>
+    </message>
+    <message>
+      <location filename="../OverrideView.ui" line="276"/>
+      <source>Memory bank controller</source>
+      <translation>Controlador de banco de memória</translation>
+    </message>
+    <message>
+      <location filename="../OverrideView.ui" line="294"/>
+      <source>MBC1</source>
+      <translation>MBC1</translation>
+    </message>
+    <message>
+      <location filename="../OverrideView.ui" line="299"/>
+      <source>MBC2</source>
+      <translation>MBC2</translation>
+    </message>
+    <message>
+      <location filename="../OverrideView.ui" line="304"/>
+      <source>MBC3</source>
+      <translation>MBC3</translation>
+    </message>
+    <message>
+      <location filename="../OverrideView.ui" line="309"/>
+      <source>MBC3 + RTC</source>
+      <translation>MBC3 + RTC</translation>
+    </message>
+    <message>
+      <location filename="../OverrideView.ui" line="314"/>
+      <source>MBC5</source>
+      <translation>MBC5</translation>
+    </message>
+    <message>
+      <location filename="../OverrideView.ui" line="319"/>
+      <source>MBC5 + Rumble</source>
+      <translation>MBC5 + Rumble</translation>
+    </message>
+    <message>
+      <location filename="../OverrideView.ui" line="324"/>
+      <source>MBC6</source>
+      <translation>MBC6</translation>
+    </message>
+    <message>
+      <location filename="../OverrideView.ui" line="329"/>
+      <source>MBC7</source>
+      <translation>MBC7</translation>
+    </message>
+    <message>
+      <location filename="../OverrideView.ui" line="334"/>
+      <source>MMM01</source>
+      <translation>MMM01</translation>
+    </message>
+    <message>
+      <location filename="../OverrideView.ui" line="339"/>
+      <source>Pocket Cam</source>
+      <translation>Pocket Cam</translation>
+    </message>
+    <message>
+      <location filename="../OverrideView.ui" line="344"/>
+      <source>TAMA5</source>
+      <translation>TAMA5</translation>
+    </message>
+    <message>
+      <location filename="../OverrideView.ui" line="349"/>
+      <source>HuC-1</source>
+      <translation>HuC-1</translation>
+    </message>
+    <message>
+      <location filename="../OverrideView.ui" line="354"/>
+      <source>HuC-3</source>
+      <translation>HuC-3</translation>
+    </message>
+    <message>
+      <location filename="../OverrideView.ui" line="359"/>
+      <source>Wisdom Tree (Unlicensed)</source>
+      <translation>Wisdom Tree (Não licenciado)</translation>
+    </message>
+    <message>
+      <location filename="../OverrideView.ui" line="364"/>
+      <source>Pokémon Jade/Diamond (Unlicensed)</source>
+      <translation>Pokémon Jade/Diamond (Sem licença)</translation>
```

Comment:
This line should be updated with "Não licenciado" to match the one before.

Reply:
I just updated this translation, thank you.